### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

**Comment by sabina-talipova** [comment]
Requires https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/33

**Comment by sabina-talipova** [comment]
Broken Behat tests are related to the issue https://github.com/silverstripeltd/product-issues/issues/655

**Review by maxime-rainville — APPROVED** [review]
LGTM

File: composer.json [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
-        "silverstripe/cms": "^4.3",
-        "silverstripe/admin": "^1.7@dev",
-        "silverstripe/framework": "^4.10"
+        "php": "^8.1",
+        "silverstripe/admin": "^2",
+        "silverstripe/cms": "^5",
+        "silverstripe/framework": "^5",
+        "silverstripe/elemental-fileblock": "^3",
+        "silverstripe/vendor-plugin": "^2",
+        "dnadesign/silverstripe-elemental": "5.0.0-alpha2"
```
Comment (michalkleiner) [inline composer.json:20]: @sabina-talipova why https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/33 requires `^5` and here we require `5.0.0-alpha2`?

Comment (sabina-talipova) [reply composer.json:20]: @michalkleiner, It's a temporary solution to pass GA with installation of lowest available version. We'll fix it after final release.
We discussed this issue [here](https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/33#issuecomment-1378191459)

**Pull request merged**